tests/test_format_unifier.py

```diff
@@ -0,0 +1,19 @@
+from utils.format_unifier import FormatUnifier
+from utils.tika_client import TikaClient
+from helpers import TEST_DATA_PATH
+import pytest
+
+format_unifier = FormatUnifier()
+
+
+def test_extract_from_pptx():
+    t = format_unifier.extract_text(TEST_DATA_PATH / "samplepptx.pptx")
+    assert "Sample PowerPoint" in t, \
+        "PowerPoint keyword should be in extracted string"
+
+
+def test_extract_from_pdf():
```

Comment:
Jeśli to wymaga kontenera uruchomionego, to powinno być oznaczone `pytest.mark.integration_test`